src/Microsoft.ServiceFabric.AspNetCore/ApplicationBuilderExtensions.cs

```diff
@@ -36,6 +36,11 @@ public static IApplicationBuilder UseServiceFabricMiddleware(this IApplicationBu
                 throw new ArgumentNullException("urlSuffix");
             }
 
+            if (urlSuffix == string.Empty)
```

Comment:
Changes to ServiceFabricSetupFilter.cs are fine as that takes care of `.UseServiceFabricIntegration(listener, ServiceFabricIntegrationOptions.None)`
Changes to `ApplicationBuilderExtensions.cs` and `ServiceFabricMiddleware.cs` will result in a breaking runtime change if user is using `UseServiceFabricMiddleware` directly.

Reply:
Good catch! It is understandable that a breaking change is not desirable here.

I have reverted the breaking changes.